egs2/slurp/slu1/local/score.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
```

Comment:
Same for here. If it is general, please put those in pyscripts

Reply:
Sure I added a symlink to pyscripts.